core-kotlin-modules/core-kotlin-6/src/main/kotlin/com/baeldung/accidental_override/Animals.kt

```diff
@@ -0,0 +1,24 @@
+package com.baeldung.accidental_override
+
+//class Cat : Animal {
+//    val sound: String = "Meow"
+//}
```

Comment:
Should we add a comment that explains the error this would create?

```suggestion
// This example will create an accidental override error, because of the conflicting `getSound()` function.
//class Cat : Animal {
//    val sound: String = "Meow"
//}
```